git_machete/cli.py

```diff
@@ -670,7 +670,7 @@ def launch(orig_args: List[str]) -> None:
             machete_client.read_definition_file(perform_interactive_slide_out=should_perform_interactive_slide_out)
             branch = get_local_branch_short_name_from_arg_or_current_branch(cli_opts.opt_branch, git)
             if branch is None or branch not in machete_client.managed_branches:
-                exit_script(1)
+                exit_script(ExitCode.ARGUMENT_ERROR)
```

Comment:
Actually, for this case I'd rather use `MACHETE_EXCEPTION` (even though it's not strictly a "machete exception")